src/api/docs/channel/types.js

```diff
@@ -21,12 +20,15 @@ export default class ChannelTypesMethod extends PaginatedMethod {
   parameters() {
     return super.parameters().concat([
       new SlugParameter(),
-      new OrderParameter(["online", "featured", "partnered", "name",
-                          "viewers_total", "followers", "subscribers"]),
-      new OnlyParameter(),
+      new OrderParameter(["online", "featured", "partnered", "name", "viewers_total", "followers", "subscribers"]),
```

Comment:
It's preferable to have things broken up onto multiple lines (usually wrapped within 79 characters)